bot/cogs/free.py

```diff
@@ -0,0 +1,89 @@
+import logging
+from datetime import datetime
+
+from discord import Colour, Embed, Member, utils
+from discord.ext.commands import BucketType, Context, command, cooldown
+
+from bot.constants import Categories, Free
+
+
+log = logging.getLogger(__name__)
+
+PYTHON_HELP_ID = Categories.python_help
```

Comment:
I don't see a point in having these module-level constants when you're importing a constant already. I'd rather see `Categories.python_help` in the class itself.